src/main/resources/worldofzuul.presentation/Solar shop.fxml

```diff
@@ -66,21 +65,20 @@
       </image>
    </ImageView>
    <Label layoutX="98.0" layoutY="220.0" text="Size: Small" textFill="WHITE" />
-   <Label layoutX="98.0" layoutY="260.0" text="Price: 2023" textFill="#fffefe" />
+   <Label fx:id="solarPrice1" layoutX="98.0" layoutY="260.0" text="Price: 2023" textFill="#fffefe" />
```

Comment:
I know this isn't the place for this, but we should really not have to hardcode the price like this, since we have a config file for that, oh well.